app/main.py

```diff
@@ -1 +1,17 @@
-# write your code here
+class InvalidCommandFormat(Exception):
+    pass
+
+
+def copy_file(command: str) -> None:
+    command = command.split()
+
+    if command[0] != "cp":
+        raise InvalidCommandFormat("Please provide a command like "
```

Comment:
After opening the parentheses, write all the arguments on a new line, including the first one, the closing bracket is also written on a new line